bootstrap_cfn/salt_utils.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python
+import sys
+sys.path.append('/usr/local/bootstrap-cfn')
```

Comment:
Hard coded paths are bad - it makes assumptions about where the code will live in multiple places in the code. I think this `os.path.dirname(__file__)` achieves what we want.

Also can you add a comment saying *why* we are doing this so we remember in 12 months time.